Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -221,25 +220,17 @@ import BraintreeCore
 
     /// Returns true if the proper Venmo app is installed and configured correctly, returns false otherwise.
     @objc public func isVenmoAppInstalled() -> Bool {
-        if let _ = application as? UIApplication {
-            guard let appSwitchURL = BTVenmoAppSwitchRedirectURL().baseAppSwitchURL else {
-                return false
-            }
-
-            return UIApplication.shared.canOpenURL(appSwitchURL)
-        } else {
-            return application.canOpenURL(BTVenmoAppSwitchRedirectURL().baseAppSwitchURL ?? URL(string: "")!)
+        guard let appSwitchURL = BTVenmoAppSwitchRedirectURL().baseAppSwitchURL else {
+            return false
         }
+        
+        return application.canOpenURL(appSwitchURL)
     }
 
     /// Switches to the App Store to download the Venmo application.
     @objc public func openVenmoAppPageInAppStore() {
         apiClient.sendAnalyticsEvent("ios.pay-with-venmo.app-store.invoked")
-        if let _ = application as? UIApplication {
-            UIApplication.shared.open(appStoreURL)
-        } else {
-            application.open(appStoreURL)
-        }
+        application.open(appStoreURL, options: [:], completionHandler: nil)
```

Comment:
Take it or leave it, but if we want to provide defaults like `open` in `UIApplication` we can provide it in a protocol extension to `URLOpener`:
```swift
public extension URLOpener {

    func open(_ url: URL, options: [UIApplication.OpenExternalURLOptionsKey: Any] = [:], completionHandler completion: ((Bool) -> Void)? = nil) {
        open(url, options: options, completionHandler: completion)
    }
}
```

Would allow us to update this and the call to open in `performAppSwitch` to:
```swift
application.open(appStoreURL) // openVenmoAppPageInAppStore
application.open(appSwitchURL) { success in // performAppSwitch
```

Reply:
Yeh this crossed my mind - but I thought the purpose of the protocol extension was confusing to read/deduce. I don't mind the passing of the empty dict and nil completion, but will defer to what folks think is more clear!

Reply:
I like the cleaner callsite but I 100% just think that's a personal preference. Lol. I am fine with either so will leave it up to you to decide!